home.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home page</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" rel="stylesheet">
+</head>
+<body>
+    <!-- First Navbar -->
+    <nav class="navbar social-navbar">
+        <ul>
+            <li><a href="https://twitter.com" target="_blank" rel="noopener"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="https://facebook.com" target="_blank" rel="noopener"><i class="fab fa-facebook"></i></a></li>
+            <li><a href="English">English</a></li>
+            <li><a href="mypage">Mypage</a></li>
+            <li><a href="Logout">Logout</a></li>
+        </ul>
+    </nav>
+    <!-- Second Navbar with Hamburger Menu -->
+<nav class="navbar main-navbar">
+    <div class="logo">
+        <a href="about.html">Global<b> Innovation</b></a>
+    </div>
+    <!-- Hamburger Icon -->
+    <div class="hamburger" id="hamburger">
+        <i class="fas fa-bars"></i>
+    </div>
+    <!-- Nav Links -->
+    <div class="nav-links-container" id="nav-links-container">
+        <ul class="nav-links" id="nav-links">
+            <li><a href="about.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="about.html">Sponsors</a></li>
+            <li><a href="#">Speakers</a></li>
+            <button id="ctaButton" class="cta-button" type="button">Innovation Schedule</button>
+        </ul>
+    </div>
```

Comment:
@Aqsamaan12  Your nav links is not working properly, as per the project requirements your nav links should work and take the users to the relevant sections.

![Screenshot 2024-10-14 131132](https://github.com/user-attachments/assets/4e019aa4-bb41-411a-babd-523a6181374c)